src/Apps/Order/Routes/__tests__/Accept.test.tsx

```diff
@@ -18,11 +18,6 @@ import {
 import { AcceptFragmentContainer } from "../Accept"
 import { OrderAppTestPage } from "./Utils/OrderAppTestPage"
 
-jest.mock("Artsy/Analytics/trackPageView", () => ({
-  trackPageView: jest.fn(),
-}))
-const mockTrackPageView = require("Artsy/Analytics/trackPageView")
-  .trackPageView as jest.Mock
```

Comment:
This test code was removed because we're now handling things one level up and no longer need individual route checks.